Makefile

```diff
@@ -55,7 +55,7 @@ test-slow-inject: og
 	# turnt --env inject_test test/*.gfa
 
 test-slow-overlap: og
-	-turnt --save --env overlap_setup test/*.gfa
+	-turnt --env overlap_setup test/*.gfa
```

Comment:
Perhaps a mistake? I think this is supposed to save its output.

Reply:
Whoops! Definitely a mistake. This has been fixed now